src/operator/roi_pooling.cc

```diff
@@ -227,15 +227,56 @@ Operator *ROIPoolingProp::CreateOperatorEx(Context ctx, std::vector<TShape> *in_
 DMLC_REGISTER_PARAMETER(ROIPoolingParam);
 
 MXNET_REGISTER_OP_PROPERTY(ROIPooling, ROIPoolingProp)
-.describe("Performs region-of-interest pooling on inputs. Resize bounding box coordinates by "
-"spatial_scale and crop input feature maps accordingly. The cropped feature maps are pooled "
-"by max pooling to a fixed size output indicated by pooled_size. batch_size will change to "
-"the number of region bounding boxes after ROIPooling")
-.add_argument("data", "NDArray-or-Symbol", "Input data to the pooling operator, a 4D Feature maps")
+.describe(R"code(Performs region of interest(ROI) pooling on the input array.
+
+ROI pooling is a variant of max pooling layer, in which output size is fixed and
+input rectangle is a parameter.  Its purpose is to perform max pooling on the inputs
```

Comment:
what precisely do you mean by "input rectangle is a parameter". How is a rectangle a parameter? What is the parameter precisely?